feat: tf runner in external config example (#17)
humanitec-architecture/example-library: resource-definitions/terraform-driver/runner/README.md

> The [Terraform Driver](https://developer.humanitec.com/integration-and-extensions/drivers/generic-drivers/terraform/) can be configured to execute the Terraform scripts as part of a Kubernetes Job execution in a target Kubernetes cluster, instead of in the Humanitec infrastructure. In this case, you must supply access to the cluster to the Humanitec Platform Orchestrator. 

Reviewer: ```suggestion
The [Terraform Driver](https://developer.humanitec.com/integration-and-extensions/drivers/generic-drivers/terraform/) can be configured to execute the Terraform scripts as part of a Kubernetes Job execution in a target Kubernetes cluster, instead of in the Humanitec infrastructure. In this case, you must supply access data to the cluster to the Humanitec Platform Orchestrator. 
```

> The examples in this section show how to provide those referencing a `k8s-cluster` resource definition as part of the [non-secret](https://developer.humanitec.com/integration-and-extensions/drivers/generic-drivers/terraform/#runner-object) and [secret](https://developer.humanitec.com/integration-and-extensions/drivers/generic-drivers/terraform/#runner-object-1) fields of the `runner` object in the Resource Definition based on the Terraform Driver.

Reviewer: ```suggestion
The examples in this section show how to provide those data by referencing a `k8s-cluster` Resource Definition as part of the [non-secret](https://developer.humanitec.com/integration-and-extensions/drivers/generic-drivers/terraform/#runner-object) and [secret](https://developer.humanitec.com/integration-and-extensions/drivers/generic-drivers/terraform/#runner-object-1) fields of the `runner` object in the `s3` Resource Definition based on the Terraform Driver.
```

* [k8s-cluster-refs.tf](k8s-cluster-refs.tf): provides a connection to an [EKS cluster](https://developer.humanitec.com/integration-and-extensions/drivers/k8-drivers/eks-cluster/).

> * [s3-ext-runner-refs.tf](s3-ext-runner-refs.tf): uses [runner](https://developer.humanitec.com/integration-and-extensions/drivers/generic-drivers/terraform/#running-the-terraform-runner-in-a-target-cluster) configuration to run the Terraform Runner in the external cluster specied by [k8s-cluster-refs.tf](k8s-cluster-refs.tf) and provision an S3 bucket. It configures the Runner to run scripts in private repositoray which initializes a [Terraform s3 backend](https://developer.hashicorp.com/terraform/language/settings/backends/s3) [https://developer.hashicorp.com/terraform/language/settings/backends/s3#credentials-and-shared-configuration](https://developer.hashicorp.com/terraform/language/settings/backends/s3#credentials-and-shared-configuration) via Environment Variables.

Reviewer: ```suggestion
* [s3-ext-runner-refs.tf](s3-ext-runner-refs.tf): uses [runner](https://developer.humanitec.com/integration-and-extensions/drivers/generic-drivers/terraform/#running-the-terraform-runner-in-a-target-cluster) configuration to run the Terraform Runner in the external cluster specified by [k8s-cluster-refs.tf](k8s-cluster-refs.tf) and provision an S3 bucket. It configures the Runner to run Terraform scripts from a private Git repository which initializes a [Terraform s3 backend](https://developer.hashicorp.com/terraform/language/settings/backends/s3) via Environment Variables.
```
